packages/core/src/actors/observable.ts

```diff
@@ -26,6 +26,45 @@ export type ObservableActorRef<TContext> = ActorRefFrom<
   ObservableActorLogic<TContext, any>
 >;
 
+/**
+ * Observable actor logic is described by an observable stream of values. Actors created from observable logic (“observable actors”) can:
+ *
+ * - Emit snapshots of the observable’s emitted value
+ *
+ * The observable’s emitted value is used as its observable actor’s `context`.
+ *
+ * Sending events to observable actors will have no effect.
+ *
+ * @param observableCreator A function that creates an observable. It receives one argument, an object with the following properties:
+ * - `input` - Data that was provided to the observable actor
+ * - `system` - The actor system to which the observable actor belongs
+ * - `self` - The parent actor
```

Comment:
```suggestion
 * - `self` - The parent actor
 * - `system` - The actor system to which the observable actor belongs
```
Switching in order of importance